DQM/GEM/plugins/GEMDigiSource.cc

```diff
@@ -26,8 +26,14 @@ void GEMDigiSource::bookHistograms(DQMStore::IBooker& ibooker, edm::Run const&,
 
   nBXMin_ = -10;
```

Comment:
`nBXMin_` and `nBXMax_` could be `const` (but not local since they are used in another method). I'll do that.

But `fRadiusMin_` and `fRadiusMax_` are more complicated. They depend on the geometry (GE11 and GE21 and ME0 have different radius ranges) and I have a plan to set these ranges from the configuration directly. So they cannot be `const`.